Classes/Views/GutterLayoutManager.swift

```diff
@@ -0,0 +1,257 @@
+//
+//  GutterLayoutManager.swift
+//  Freetime
+//
+//  Created by Weyert de Boer on 15/10/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+protocol GutterLayoutManagerDelegate: class {
+
+    func getLineForLine(gutterLayoutManager: GutterLayoutManager, lineNumber: Int, for line: String) -> String
```

Comment:
Possible to rename this delegate method to something that reads a bit better and makes more sense immediately?

Reply:
If you have any suggestions 👼 What about `getVisualLineNumberForGutter(gutterLayoutManager: GutterLayoutManager, lineIndex: Int, for line: String)`?